core/src/smartcontracts/wasm.rs

```diff
@@ -251,6 +255,21 @@ pub fn load_module(engine: &Engine, bytes: impl AsRef<[u8]>) -> Result<wasmtime:
     Module::new(engine, bytes).map_err(Error::ModuleLoading)
 }
 
+pub(crate) fn serialize_module_base64<S>(module: &Module, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: serde::Serializer,
+{
+    let bytes = module.serialize().map_err(serde::ser::Error::custom)?;
+    let wrapper = Base64Wrapper(bytes);
+    wrapper.serialize(serializer)
+}
+
+/// Deserialize [`Module`]. Accepts only output of [`Module::serialize`].
+#[allow(unsafe_code)]
+pub(crate) fn deserialize_module(engine: &Engine, bytes: impl AsRef<[u8]>) -> Result<Module> {
+    unsafe { Module::deserialize(engine, bytes).map_err(Error::ModuleDeserialization) }
```

Comment:
I think you should put a comment here. Why is it acceptable to use this unsafe method?

Reply:
Added